pep-0584.rst

```diff
@@ -1,1043 +1,974 @@
 PEP: 584
-Title: Add + and += operators to the built-in dict class.
+Title: Add Union Operators To dict
 Version: $Revision$
 Last-Modified: $Date$
 Author: Steven D'Aprano <steve@pearwood.info>,
         Brandt Bucher <brandtbucher@gmail.com>
-Status: Draft
+BDFL-Delegate: Guido van Rossum <guido@python.org>
+Status: Final
```

Comment:
No the status is not Final. It's still Draft. I haven't accepted this draft and even if I had the Steering Council would have to agree first.

Reply:
Ah, I misunderstood the term "Final" here. The flowchart in PEP 1 is useful.